src/js/video.js

```diff
@@ -147,6 +148,15 @@ videojs.hooks_ = {};
  */
 videojs.hooks = function(type, fn) {
   videojs.hooks_[type] = videojs.hooks_[type] || [];
+  if (fn && type === 'vttjsloaded') {
+    vttjsOnLoad((vttjs) => {
+      const vttjsLoadedHooks = videojs.hooks('vttjsloaded');
+
+      vttjsLoadedHooks.forEach((hookFn) => hookFn(vttjs));
+      // we notified the listeners, now clear them out
+      vttjsLoadedHooks.length = 0;
```

Comment:
Yup, it's to make sue that each of the `vttjsloaded` hooks is only ever called the one time.